examples/duplex_s2s/s2s_duplex_train.py

```diff
@@ -19,7 +19,7 @@
 from omegaconf import OmegaConf
 
 from nemo.collections.duplex_s2s.data.datamodule import S2SDataModule
```

Comment:
I think would be nice if we could create a s2s_duplex_speech_decoder_train.py and keep s2s_duplex_train.py file for INTERSPEECH-like architecture.